src/gui/editors/PianoRoll.cpp

```diff
@@ -2811,8 +2811,8 @@ void PianoRoll::dragNotes( int x, int y, bool alt, bool shift, bool ctrl )
 				if (note->selected())
 				{
 					int newLength = note->oldLength() + off_ticks;
-					newLength = qMax(1, newLength);
-					note->setLength( MidiTime(newLength) );
+					if (newLength <= 0) { newLength = alt ? 1 : quantization(); }
+					note->setLength(MidiTime(newLength));
```

Comment:
This works the way I wanted, judging from an admittedly quick test.
```suggestion
					int minLength = alt ? 1 : (note->oldLength() % quantization());
					if (minLength == 0) { minLength = quantization(); }
					note->setLength(MidiTime(max(newLength, minLength)));
```

Would love to figure out a shorter way to get 
`x % y` if `x % y > 0`
`y` otherwise

Reply:
It works nice but if I set the length of notes of varying length I would want them to end up at the same min value. Example two notes starting at the same time. Lenths, 2/16 and 3/16. Set quantization to 1/8, hit Ctrl + A and set their length to min by dragging. Not the same value.

Reply:
In that case I think we need to iterate over the selction once before getting here to find the smallest minLength of any selected note, then use that for all of them.

Reply:
Had to test my latest research so PR is for now back at my branch.